Packs/Expanse/Integrations/Expanse/Expanse.py

```diff
@@ -914,6 +957,116 @@ def exposures_command():
     return_outputs(human_readable, ec, exposures)
 
 
+def domains_for_certificate_command():
+    """
+
+    :return:
+    """
+    search = demisto.args()['common_name']
+    params = {
+        "commonNameSearch": search
+    }
+    token = do_auth()
+
+    observed_ips = []
+    matching_domains = []
+
+    certificates = _fetch_certificates(params=params, token=token)
+    for certificate in certificates:
+        certificate_details = _fetch_certificate(md5_hash=certificate['certificate']['md5Hash'], token=token)
+        for ip in certificate_details['details']['recentIps']:
+            observed_ips.append(ip['ip'])
+
+    for ip in observed_ips:
+        params = {
+            'inetSearch': ip,
+            'assetType': 'DOMAIN'
+        }
+        matching_domains += _fetch_ips(params=params, token=token)
+
+    if len(matching_domains) == 0:
+        demisto.results("No data found")
+        return
+
+    context = get_expanse_certificate_to_domain_context(common_name=search, data=matching_domains)
+
+    ec = {
+        'Expanse.IPDomains(val.SearchTerm == obj.SearchTerm)': context
+    }
+
+    raw_domain_objects = context['DomainList']
+    del context['DomainList']  # Remove full objects from human readable response
+    human_readable = tableToMarkdown("Expanse Domains matching Certificate Common Name: {search}".format(search=search),
+                                     context)
+    context['DomainList'] = raw_domain_objects
```

Comment:
the logic here for building the human readable and context outputs here is weird
no need to remove the domain list from context and add it again, just initialize a list for the humanreadable while will be a copy of the context and remove from it the domainlist

Reply:
Good call, there was a better way to do this - I'd just pulled from an earlier method.